design/workflow.md

```diff
@@ -54,14 +54,14 @@ jobs:
 
 ### SCM Branch-specific jobs
 
-Use a regex filter after `~commit:` to denote a job only run when code changes are made to branches matching that pattern (eg: `/^staging$/`, `/^user-.*$/`).
+Use exact match of a string (eg: `staging`) or a slash-quoted regex filter (eg: `/user-.*/`) after `~commit:` to denote a job only run when code changes are made to branches matching that pattern　(eg: `staging`, `/^user-.*$/`).
```

Comment:
Think "slash-quoted" is redundant here, can remove it and just say "regex filter". 😄

Reply:
Thanks I will fix!